tox.ini

```
@@ -1,42 +1,38 @@
-[pytest]
-filterwarnings =
-    # xdist/plugin.py:227
-    ignore:The --rsyncdir .* are deprecated:DeprecationWarning
+[tox]
+envlist =
+    docs
+    pre-commit
+    python3.{11, 10, 9, 8, 7}
+skip_missing_interpreters = true
+minversion = 3.24.5
+isoleted_build = true
```

```suggestion
isolated_build = true
```
Thanks to @mborsetti :)